### ISSUE TITLE
Dump failure with error code:3

### ISSUE THREAD

**Issue by EinPakistein** [EKA2L1/Dumber#5]
after installing Dumberdore_S60_5_X_v_1_0_0.sis in my N86 phone, i can't find "RnD tools" in main meun, also can't run Dumberdore.exe under C:/sys/bin/, it complain "can't open"
I also tried open Dumberdore app, and click "Dump RPKG", it show "Calculate dump size" firstly, then show "Dump failure with error code:3" error message.
How can i dump RPKG successfully?

**Comment by pent0** [comment]
Yes, sorry for the trouble. I'm resolving it and will give you result as soon as possible! 

 I also have updated the README to reflect what to do on both the wiki and here.

**Comment by EinPakistein** [comment]
Thank you very much, no rush! will try your fixing after done!

**Comment by pent0** [comment]
Hi, I have investigated and it looks like you need to enable the Patch
Open4All in RomPatcherPlus for the dumper to work. Please help me by try
enabling it! If you got success with it then I will add a note in the dump.

Thank you very much!

Vào Th 3, 6 thg 10, 2020 vào lúc 16:32 EinPakistein <
notifications@github.com> đã viết:

**Comment by EinPakistein** [comment]
Hi, I reinstalled the latest Dumberdore sis/sisx, and enabled "Open4All" in RompatcherPlus.
it shown "Calculate dump size" firstly, then shown "Dump failure with error code:3 (category KERN-EXEC)!" after several seconds.

**Comment by pent0** [comment]
Yes, it seems Open4All is not all the root cause. I will continue
investigating, but it's very hard to know why without a debugger, so it
might take sometimes. Thank you.

Vào Th 5, 8 thg 10, 2020 vào lúc 16:47 EinPakistein <
notifications@github.com> đã viết:

**Comment by EinPakistein** [comment]
Thank you very much!
I have another question: For now, every user need to dump rom and RPKG on phone by themselves, this look abit complex. Can we share the usable roms and RPKGs on website? or integrate them into EKA2L1 software directly?

**Comment by pent0** [comment]
That's not something I can answer, since it's questionable about legality.
Sorry.

Vào Th 7, 10 thg 10, 2020 vào lúc 16:52 EinPakistein <
notifications@github.com> đã viết:

**Comment by pent0** [comment]
I pushed a new release, integrated from @artem78 . Please check out. If you have the same issue as before, I hope you can help us by setting up a debugger. I will guide you about that when it is needed. Sorry for the inconvenience.

**Comment by EinPakistein** [comment]
Thank you very much! i'd like to try the latest version and set up debugger if need.
But i can't install the new sis/sisx, i tried to install it, but it required network... for now i have no SIM card for this N86 phone. So i can't install it now.

**Comment by pent0** [comment]
Can you try copying the SISX from PC to your phone? The SIS dont require any network, so it's weird it requests so. Screenshot me if you can!

**Comment by EinPakistein** [comment]
[Dumber issue snapshots.zip](https://github.com/EKA2L1/Dumber/files/5392994/Dumber.issue.snapshots.zip)
Hi I uploaded a video for this strange issue, we can see that after i click open botton, the network selection widwon pop up...
I also updated 2 picture about old version issue.

**Comment by artem78** [comment]
Do you really try to open file with `sis` extension? Looks weird.

**Comment by artem78** [comment]
Try to install it using Nokia PC Suite.

------- Первоначальное сообщение -------

**Comment by ericxel** [comment]
can someone tell me how to create a blank site and when you open it wait 5 seconds and then open google ?PLS IM LITTERALY 13 YO
